docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -518,17 +559,17 @@ Your module's motor direction defines, in part, how your module will interpret a
 
 Motor direction is enumerated as:
 
-0. Unconfigured
-1. 3D Counter Clockwise
-2. 3D Clockwise
-3. 2D Counter Clockwise
-4. 2D Clockwise 
+1. Unconfigured
```

Comment:
Wait is this right? Or was this an overeager formatter?
![image](https://github.com/user-attachments/assets/6e464f5d-2ee8-4858-841f-1bee33e5957e)

Reply:
ughh I fixed this twice already. vscode thinks it's smarter than I am.